app/(gcforms)/[locale]/(form administration)/form-builder/[id]/preview/Preview.tsx

```diff
@@ -42,15 +43,14 @@ export const Preview = ({
     getSecurityAttribute: s.getSecurityAttribute,
   }));
 
-  // TODO probably redirecting to the error page makes more sense since the error is not recoverable
-  const formParsed = safeJSONParse(getSchema());
-  if (formParsed?.error) {
+  const formParsed = safeJSONParse<FormProperties>(getSchema());
+  if (!formParsed) {
     toast.error(<ErrorSaving errorCode={FormServerErrorCodes.JSON_PARSE} />, "wide");
   }
 
   const formRecord: PublicFormRecord = {
     id: id || "test0form00000id000asdf11",
-    form: formParsed,
+    form: formParsed || defaultForm, // defaultForm to stop a redirect to the app error page
```

Comment:
Or perhaps it would be better to remove `defaultForm` and let an error be thrown and the app redirect to the error page. This way the user sees a correct error, not very helpful but correct. Versus handling the error but showing an unhelpful error.